keyboards/enviousdesign/delirium/rev0/info.json

```diff
@@ -0,0 +1,121 @@
+{
+    "manufacturer": "Envious.Design",
+    "keyboard_name": "Delirium Prototype",
+    "maintainer": "Envious.Design",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15", "GP19", "GP18", "GP17", "GP16"],
+        "rows": ["GP20", "GP21", "GP22", "GP26", "GP27", "GP28"]
+    },
+    "processor": "RP2040",
+    "url": "http://envious.design",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0xA100",
+        "vid": "0xE739"
+    },
+    "community_layouts": ["tkl_iso"],
+    "layouts": {
```

Comment:
does this version of delirium have a `tkl_ansi` layout?

Reply:
This version does not have ANSI support.